tests/acceptance/file-details/dynamic-scan-test.js

```
@@ -520,6 +520,8 @@ module('Acceptance | file-details/dynamic-scan', function (hooks) {
 
       assert.dom(scanStartBtn).doesNotExist();
 
+      debugger;
```

Biome

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)